runtime/moonbase/src/lib.rs

```diff
@@ -522,6 +522,44 @@ impl pallet_treasury::Config for Runtime {
 	type SpendFunds = ();
 }
 
+parameter_types! {
+	// Add one item in storage and take 258 bytes
+	pub const BasicDeposit: Balance = currency::deposit(1, 258);
+	// Not add any item to the storage but takes 66 bytes
+	pub const FieldDeposit: Balance = currency::deposit(0, 66);
+	// Add one item in storage and take 53 bytes
+	pub const SubAccountDeposit: Balance = currency::deposit(1, 53);
```

Comment:
The comments on these are a little confusing to me. I looked up the corresponding comments in the identity pallet:

```rust
        /// The amount held on deposit for a registered identity
        #[pallet::constant]
        type BasicDeposit: Get<BalanceOf<Self>>;

        /// The amount held on deposit per additional field for a registered identity.
        #[pallet::constant]
        type FieldDeposit: Get<BalanceOf<Self>>;

        /// The amount held on deposit for a registered subaccount. This should account for the fact
        /// that one storage item's value will increase by the size of an account ID, and there will be
        /// another trie item whose value is the size of an account ID plus 32 bytes.
        #[pallet::constant]
        type SubAccountDeposit: Get<BalanceOf<Self>>;
```

So does this mean we're charging only 258/66/53 wei for these? That doesn't seem sufficient to prevent spam to me.

Reply:
It's not wei. Look at the defn of `fn deposit`:

```rust
pub const MICROUNIT: Balance = 1_000_000_000_000;
..
pub const UNIT: Balance = 1_000_000_000_000_000_000;
..
pub const STORAGE_BYTE_FEE: Balance = 100 * MICROUNIT;

pub const fn deposit(items: u32, bytes: u32) -> Balance {
	items as Balance * 1 * UNIT + (bytes as Balance) * STORAGE_BYTE_FEE
}
```